hydra/core/utils.py

```diff
@@ -126,11 +128,19 @@ def run_job(
             _save_config(config.hydra.overrides.task, "overrides.yaml", hydra_output)
 
         with env_override(hydra_cfg.hydra.job.env_set):
+            Callbacks.instance().on_job_start(
+                config=config,
+            )
             ret.return_value = task_function(task_cfg)
         ret.task_name = JobRuntime.instance().get("name")
 
         _flush_loggers()
 
+        Callbacks.instance().on_job_end(
+            config=config,
+            job_return=ret,
+        )
```

Comment:
No reason for this to spawn 4 lines, right? if you remove the trailing comma it would probably fit in one line.

Reply:
true, fixed this and other places as well.